### ISSUE TITLE
Permissions not working fine on Android P

### ISSUE THREAD

**Issue by miguelbcr** [miguelbcr/RxPaparazzo#100]
As you can see in the screenshot if you open the app delegating the permissions to [RxPermissions](https://github.com/tbruyelle/RxPermissions), in spite of granting the permissions correctly a "Permission Denied" exception is thrown when picking an image up. But if you kill the app and repeat the process it works fine 🤔 

![rxpaparazzo-androidP-permissions](https://user-images.githubusercontent.com/2734557/54073461-a6ebcc00-4287-11e9-9974-09e2284c9842.gif)

On the other hand if you grant the permissions directly from the app's settings and then launch the app it works fine

![rxpaparazzo-androidP-permissions-1](https://user-images.githubusercontent.com/2734557/54073466-abb08000-4287-11e9-84ff-5a4da23af702.gif)

**Comment by ahmed3elshaer** [comment]
I think I had the same issue and when I tried to show the exception cause I got this 
```
java.io.FileNotFoundException: /storage/emulated/0/RxPaparazzo/SAVED-20190808_0444_11666.jpg: open failed: EACCES (Permission denied)
```
but it works fine in the second runtime

@miguelbcr  are you using emulator too?
cuz I just run this on a physical device - Android P - and it didn't happen

**Comment by miguelbcr** [comment]
> @miguelbcr are you using emulator too?

yeah, emulator. I had no chance of checking it on a real device 😕 
Thanks for your comment!

**Comment by ahmed3elshaer** [comment]
> > @miguelbcr are you using emulator too?
> 
> yeah, emulator. I had no chance of checking it on a real device 😕
> Thanks for your comment!

Anytime! 
try on a physical device and let me know the results

**Comment by miguelbcr** [comment]
Sure! thanks!

**Comment by wodzu96** [comment]
Unfortunately I have the same issue as @ahmed3elshaer on physical device (Pixel 3). Please let me know if you are going to fix it in the near future? If you want any additional info feel free to ask.
Thanks!

**Comment by galmax1** [comment]
<img width="405" src="https://user-images.githubusercontent.com/56034696/133229118-e9540449-0f67-42f7-b186-141912632782.png">

You can try .useInternalStorage(). In my case it works!
But the image will not be available in the device gallery.
